typescript typings

When I try to use

```
import { arcgisToGeoJSON } from "@terraformer/arcgis";
```

in a TypeScript project, I get the error:

```
semantic error TS7016: Could not find a declaration file for module '@terraformer/arcgis'.
```

How can we make these available? I think there was a [previous version](https://www.npmjs.com/package/@types/arcgis-to-geojson-utils), perhaps we can use those?

--- a/packages/arcgis/README.md
+++ b/packages/arcgis/README.md
@@ -114,6 +114,14 @@ Terraformer.geojsonToArcGIS(/* ... */);
 
 ## [Contributing](./CONTRIBUTING.md)
 
+## TypeScript
+
+Type definitions for `@terraformer/arcgis` can be found at [@types/terraformer__arcgis](https://www.npmjs.com/package/@types/terraformer__arcgis). To install into your own project:
+
+```
+npm install @types/terraformer__arcgis
+```
+
 ## Ports
 
 | Project | Language | Status | Maintainer |

